roles/monitoring_bbb/defaults/main.yml

```diff
@@ -15,6 +15,7 @@ bbb_turn_server_exporter_enabled: true
 bbb_node_exporter_basic_auth_secret_name: "{{ bbb_instance_name }}-node-exporter-basic-auth"
 # Name of the Kubernetes secret containing the IONOS credentials
 bbb_ionos_basic_auth_secret_name: "{{ bbb_instance_name }}-ionos"
+bbb_ionos_basic_auth_secret_key: password
```

Comment:
Is this correct? not "{{ password }}" ?

Reply:
Yes, this is the name of the field in the Kubernetes Secret where the password is stored